### DISCUSSION TITLE
rock 5b images provided in this not booting on the 5b board. I tried cli, xfc  with sd card / balena...

### DISCUSSION THREAD

**Discussion by hemsinghb** [Joshua-Riek/ubuntu-rockchip#302] [General]
rock 5b images provided in this site not booting on the 5b board. I tried cli, xfc  with sd card / balena.... led on the board just flickring  continuously... thats it.

**Comment by Joshua-Riek** [comment]
Use the previous release, the bootloader firmware has an issue with the rock 5 boards, which I'm trying to fix now.

**Reply by Joshua-Riek** [reply]
Wait, if the led is blinking, everything should be working fine. Check your HDMI connection or connect a serial console to your board.

**Reply by hemsinghb** [reply]
Now the previous image i.e.   focal 20.04 version originally supplied bu radxa also not booting. I have erased all boot loader which i have installed.    { new sd card also used. }

**Reply by Joshua-Riek** [reply]
Are you using a 5v 4A power supply? Does the previous version work?
https://github.com/Joshua-Riek/ubuntu-rockchip/releases/tag/v1.21
